samtranslator/model/preferences/deployment_preference_collection.py

```diff
@@ -1,9 +1,9 @@
-from deployment_preference import DeploymentPreference
+from .deployment_preference import DeploymentPreference
```

Comment:
You should add `from __future__ import absolute_import`, which will make this fail on python 2 as well